src/components/DisplayResults.js

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from "react";
+import { QuizContext } from "../reducers/quizReducer";
+import { businessData } from "../data/businessData";
+import { iplData } from "../data/iplData";
+import { question as suitsData } from "../data/suitsData";
+
+const DisplayResults = ({
+  data,
+  suitsCorrectAnswers,
+  businessCorrectAnswers,
+}) => {
+  const { quizState, dispatch } = useContext(QuizContext);
+  console.log(data);
+  return (
+    <section className="results-container">
+      {data.map(({ question, correctAnswer, options }) => {
+        return (
+          <div key={question} className="result-quiz-answer-container-card">
+            <p>{question}</p>
+            <div className="result-quiz-answer-option-container">
+              {options.map((item) => {
+                return (
+                  <button
+                    key={item}
+                    className={
+                      correctAnswer.includes(item)
```

Comment:
have a look at [classnames library](https://www.npmjs.com/package/classnames/), it will help you improve readability of your code